tests/test_mixed_simulator_device.py

```diff
@@ -469,16 +469,16 @@ def test_basis_state_not_analytic_error(self, simulator_device_1_wire):
             simulator_device_1_wire.apply([qml.BasisState(np.array([0]), wires=[0])])
 
     def test_qubit_state_vector_not_analytic_error(self, simulator_device_1_wire):
-        """Tests that application of QubitStateVector raises an error if the device
+        """Tests that application of StatePrep raises an error if the device
```

Comment:
Should we still keep the tests for `QubitStateVector` as well until we deprecate it